sdcm/fill_db_data.py

```diff
@@ -3086,6 +3098,7 @@ def get_scylla_version(self):
         # NOTE: node.get_scylla_version() returns following structure of a scylla version:
         #       4.4.1-0.20210406.00da6b5e9
         #       And 'parse_version' behaves differently for full and short scylla versions
+        #       And 'parse_version' behaves differently for full and short scylla versions
```

Comment:
Looks like mistakenly added copy of above line.

Reply:
Thanks, removed